onnxscript/converter.py

```diff
@@ -443,8 +443,11 @@ def translate_attr(self, attr_name, expr):
 
                 # Create GraphProto attribute
                 val = val.to_graph_proto()
-            return self.ir_builder.make_attr(attr_name, val)
-        return self.ir_builder.make_attr(attr_name, self.eval_constant_expr(expr))
+        else:
+            val = self.eval_constant_expr(expr)
+        if val is None:
```

Comment:
Would it be helpful to add a comment on the case this is handling?

Reply:
Done